### ISSUE TITLE
Translation files of es.upv.paella.opencast.episodesFromSeries is not added to code base from Crowdin

### ISSUE THREAD

**Issue by Gao-Jun** [opencast/opencast#2945]
`modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.episodesFromSeries/localization/en-US.json` is added to Crowdin with path `9.x/modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.episodesFromSeries/localization/en-US.json`.
Yet translation files of it are not included in the code base during automatically update process.

Branches such as `stable`, `r/10.x`, `r/9.x` all doesn't include corresponding translation files.

I think these files should be included when translation files update.

**Comment by lkiesow** [comment]
I think I know what's happening and can easily fix this:

Our integration will not update translation files unless they have been added once, since the people introducing Crowding years ago added lots of languages which have never been properly translated and which we did not want to show in the interfaces. I guess someone just missed the new files. Maybe we could create a master list instead and add all language files once they are listed in there. For now, let's see if I can add the missing files.
